pandamium_datapack/data/pandamium/functions/misc/global_sleeping.mcfunction

```diff
@@ -1,8 +1,7 @@
-
 scoreboard players set <sleeping_divisor> variable 3
-execute store result score <overworld_player_count> variable if entity @a[nbt={Dimension:"minecraft:overworld"}]
```

Comment:
How is `[x=0]` the same as `[nbt={Dimension:"minecraft:overworld"}]`?

Reply:
Setting x/y/z sets the position to start searching from, used by dx/dy/dz/distance. It doesn't do anything else, except that only the entity/player list of the current dimension is selected.